src/pages/index.html

```diff
@@ -101,11 +101,16 @@ <h3 class="title is-3 is-spaced is-title-reveal">How it works</h3>
     </div>
 </section>
 
-<section id="table" class="section section-light-grey is-medium">
+<section id="load-balancers" class="section section-light-grey is-medium">
     <div class="container">
             <div class="title-wrapper has-text-centered">
                 <h2 class="title is-2">Load balancers</h2>
                 <div class="divider is-centered"></div>
+
+                <p class="pt-20">
+                   List of publicly available Vedran load balancers. If you are hosting Vedran load balancer,
```

Comment:
except the typo for instructions I would rephrase this: `If you are hosting Vedran load balancer and want to be on this list, see the editing instructions.`